config/env/env.go

```diff
@@ -19,6 +19,7 @@ const HTTPClientMaxIdleConnsPerHost = "SPIKE_HTTP_CLIENT_MAX_IDLE_CONNS_PER_HOST
 const HTTPClientResponseHeaderTimeout = "SPIKE_HTTP_CLIENT_RESPONSE_HEADER_TIMEOUT"
 const HTTPClientTimeout = "SPIKE_HTTP_CLIENT_TIMEOUT"
 const HTTPClientTLSHandshakeTimeout = "SPIKE_HTTP_CLIENT_TLS_HANDSHAKE_TIMEOUT"
+const HTTPServerReadHeaderTimeout = "SPIKE_HTTP_SERVER_READ_HEADER_TIMEOUT"
```

Comment:
This has a sane default, and ideally should not be meddled with; but we are still giving the user the flexibility.